pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -355,27 +360,17 @@ def _wait_for_search_list(driver):
             _wait_for_search_list, msg="No values found in SingleSelect search"
         )
 
-    def is_editable(self):
+    def is_editable(self) -> bool:
```

Comment:
cannot it be simplified to
```python
return self.searchable or self.allow_new_values
```
I try to think another scenario when we have searchable single select non editable or combobox that allows new values non editable. Do you know any?